syntax = "proto3";

package suggestions;

service SuggestionsService {
    rpc GetSuggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
    User user = 1;
}

message SuggestionsResponse {
    repeated Book suggested_books = 1;
}

message User {
    string name = 1;
    string email = 2;
}

message Book {
    string bookId = 1;
    string title = 2;
    string author = 3;
}